icons/hidden.svg

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What license does this svg has? I also don't like the asymmetry of it (see https://pub.bilke.org/2017-02-28-CPditCOlGAc/hidden-fix.png)